supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -295,7 +295,7 @@
 // RHSSO: Added "In Red Hat Single Sign-On,"
 [[client]]
 ==== image:images/yes.png[yes] client
-*Description*: In Red Hat Single Sign-On, a _client_ is an entity that can request Red Hat Single Sign-On to authenticate a user. Most often, clients are applications and services that want to use Red Hat Single Sign-On to secure themselves and provide a single sign-on solution. Clients are also entities that request identity information or an access token so that they can securely invoke other services on the network that are secured by Red Hat Single Sign-On.
+*Description*: 1) Generically, a _client_ is an entity, such as a machine, process, service, or application) that uses a service provided by a server. Avoid using "client" to refer to an application that manages configuration of such clients, or to applications, tools, and so on. 2) In Red Hat Single Sign-On, a _client_ is an entity that can request Red Hat Single Sign-On to authenticate a user. Most often, clients are applications and services that want to use Red Hat Single Sign-On to secure themselves and provide a single sign-on solution. Clients are also entities that request identity information or an access token so that they can securely invoke other services on the network that are secured by Red Hat Single Sign-On.
```

Comment:
```suggestion
*Description*: _Client_ has multiple meanings, depending on the context:

* Generically, a _client_ is an entity, such as a machine, process, service, or application, that uses a service provided by a server. Avoid using _client_ to refer to an application that manages configuration of such clients or to refer to applications, tools, and so on.

* In Red Hat Single Sign-On, a _client_ is an entity that can request Red Hat Single Sign-On to authenticate a user. Most often, clients are applications and services that want to use Red Hat Single Sign-On to secure themselves and provide a single sign-on solution. Clients are also entities that request identity information or an access token so that they can securely invoke other services on the network that are secured by Red Hat Single Sign-On.
```

- I would suggest breaking these two items into a bulleted list, rather than a numbered list in a single paragraph. A numbered list implies that the two items are steps in procedure even though they are not.
- Replace closing parenthesis with a comma in item 1.
- Italicize the word being defined.